### PR TITLE
Fixes #23575 - avoid change of heights on fact modal

### PR DESCRIPTION
**Pull request by lizagilman**
before:

image

after:

image

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/factCharts/index.js [hunk 1]
```diff
@@ -88,7 +88,9 @@ class FactChart extends React.Component {
               </Modal.Title>
             </Modal.Header>
             <Modal.Body>
+              <div id="factChartModalBody">
               <Loader status={factChart.loaderStatus}>{[chart, error]}</Loader>
```
Comment (boaz0) [inline webpack/assets/javascripts/react_app/components/factCharts/index.js]: nitpick: can you indent `<Loader>` in between `<div>` and `</div>`?

**Comment by ohadlevy** [comment]
just one small comment, if you use redux you don't need to stop it in a breakpoint to get the image, just use the time machine slider to get to the right state (or fire your own action with the pending status) :)

**Comment by ohadlevy** [comment]
@amirfefer @sharvit can you review please?

**Review by boaz0 — COMMENTED** [review]
LGTM.

**Comment by tbrisker** [comment]
Looks like your editor is automatically making some style changes to the files when you save, such as changing single to double quotes. Do we have an agreed style guide for these things? If so, lets make sure everyone's editor uses the same style and enforce it with hound, so that we don't have these styles change back and forth depending on who last edited the file. (It also makes it more difficult to notice what is the actual change and what is just auto-styling)

**Comment by lizagilman** [comment]
I think eslint / `eslint --fix ` takes care of that. (I ran it before I pushed, this pr should be ok). 
@sharvit can you confirm?

**Comment by sharvit** [comment]
@tbrisker we do have agreed style guide, see https://github.com/theforeman/foreman/blob/develop/.eslintrc
(we want to migrate to patternfly-react-eslint)

We are running the hound eslint only on modified or new files so when @lizagilman changed this file, she had to fix the linting as she did :+1:

**Comment by tbrisker** [comment]
@sharvit great, just wanted to make sure this doesn't get changed back when someone else edits the file :smiley_cat:

**Comment by ohadlevy** [comment]
@sharvit are you sure? I'm not aware we changed from single quotes (')to
double (") ?

**Comment by sharvit** [comment]
@ohadlevy It's duble-quotes only in jsx :)

**Review by sharvit — APPROVED** [review]
LGTM thanks @lizagilman :+1:

**Review by ohadlevy — APPROVED** [review]

**Pull request merged**